backend/alert/urls.py

```diff
@@ -13,8 +13,7 @@
 
 urlpatterns = [
     path("", views.index, name="index"),
-    path("courses/", courses.views.SectionList.as_view()),
-    path("statusupdate/<slug:full_code>/", StatusUpdateView.as_view()),
```

Comment:
Was statusupdate part of the 3rd party api that's getting removed soon?

Reply:
statusupdate was included in the url routes for both `alert` and `courses`. Just removed it from `alert` so it only lives in one place.